gulpfile.babel.js

```diff
@@ -33,11 +33,13 @@ import zip from 'gulp-zip';
 
 // Import theme-specific configurations.
 var config = require('./dev/config/themeConfig.js');
+var themeSlug = config.theme.slug;
```

Comment:
We want to take into account slug and name, as either or both can change.  If either changes, then we want to trigger a rebuild process.  Therefore, I'd change this line of code to:

```
var themeConfig = config.theme;
themeConfig.isFirstRun = true;
```

The advantage here is that we capture the initial theme's configuration.  We'll use this to modify it as we run the php task..

The `themeConfig.isFirstRun` property is capturing the initial "on script load" occurrence.  That way we can run the php function when the first first loads.  It's a cheap way to note that the script is first running.